config/ConfigExample/RemoteConfigViewController.swift

```diff
@@ -48,41 +49,41 @@ class RemoteConfigViewController: UIViewController {
 
   // MARK: - Firebase 🔥
 
+  // Add additional keys that are in the console to see them in the log after fetching the config.
+  struct QSConfig: Codable {
+    let topLabelKey: String
+    let bottomLabelKey: String
+    let freeCount: Int
+  }
+
   /// Initializes defaults from `RemoteConfigDefaults.plist` and sets config's settings to developer mode
   private func setupRemoteConfig() {
     remoteConfig = RemoteConfig.remoteConfig()
-    remoteConfig.setDefaults(fromPlist: "RemoteConfigDefaults")
+    // This is an alternative to remoteConfig.setDefaults(fromPlist: "RemoteConfigDefaults"))
+    try? remoteConfig.setDefaults(from: QSConfig(topLabelKey: "myTopLabel",
+                                                 bottomLabelKey: "Buy one get one free!",
+                                                 freeCount: 4))
 
     let settings = RemoteConfigSettings()
     settings.minimumFetchInterval = 0
     remoteConfig.configSettings = settings
   }
 
-  /// Fetches remote config values from the server
-  private func fetchRemoteConfig() {
-    remoteConfig.fetch { status, error in
-      guard error == nil else { return self.displayError(error) }
-      print("Remote config successfully fetched!")
-    }
-  }
-
-  /// Activates remote config values, making them available to use in your app
-  private func activateRemoteConfig() {
-    remoteConfig.activate { success, error in
-      guard error == nil else { return self.displayError(error) }
-      print("Remote config successfully activated!")
-      DispatchQueue.main.async {
-        self.updateUI()
-      }
-    }
-  }
-
   /// Fetches and activates remote config values
   @objc
   private func fetchAndActivateRemoteConfig() {
-    remoteConfig.fetchAndActivate { status, error in
-      guard error == nil else { return self.displayError(error) }
+    Task.init {
+      guard let _ = try? await remoteConfig.fetchAndActivate() else {
+        print("Failed to fetchAndActivate.")
+        return
+      }
```

Comment:
nit: catch the error and use `displayError` to maintain parity with the old code

Reply:
Good catch!